Resources/Prototypes/_DV/Entities/Structures/Machines/lathe.yml

```diff
@@ -11,3 +11,164 @@
     - BorgModulesSyndicateStatic
   - type: Machine
     board: SyndieExosuitFabricatorMachineCircuitboard
+
+- type: entity
+  parent: BaseLatheLube
+  id: EngineeringTechFab
+  name: engineering techfab
+  description: Prints equipment for engineers.
+  components:
+  - type: Sprite
+    sprite: Structures/Machines/techfab.rsi
+    layers:
+    - state: icon
+      map: ["enum.LatheVisualLayers.IsRunning"]
+    - state: engi
+    - state: unlit
+      shader: unshaded
+      map: ["enum.PowerDeviceVisualLayers.Powered"]
+    - state: inserting
+      map: ["enum.MaterialStorageVisualLayers.Inserting"]
+    - state: panel
+      map: ["enum.WiresVisualLayers.MaintenancePanel"]
```

Comment:
make a base techfab with the sprite path + Lathe idleState/runningState

Reply:
how does that work w/ overriding only a single layer to change which fab it is

Reply:
cant do layers, just do the rsi and lathe states